spring-xd-dirt/src/main/resources/META-INF/spring-xd/global/parent-context.xml

```diff
@@ -12,4 +12,8 @@
 		<bean id="jsonPropertyAccessor" class="org.springframework.integration.json.JsonPropertyAccessor"/>
 	</int:spel-property-accessors>
 
+	<!-- This is a workaround until SI supports newer JsonPath. See INT-3624-->
+	<int:spel-function id="jsonPath" class="org.springframework.xd.dirt.integration.json.NewJsonPathUtils"
+					   method="evaluate(java.lang.Object, java.lang.String, com.jayway.jsonpath.Predicate[])" />
```

Comment:
No need to specify whole method signature since you have only a single method in the `NewJsonPathUtils`.
It will work even if you have several methods, but with different names.
We need signature only if we have overloaded methods.